tools/integration_tests/run.py

```diff
@@ -13,39 +14,69 @@
 DEBUG_BUILD = "target/debug/syncserver"
 RELEASE_BUILD = "/app/bin/syncserver"
 
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Run the syncstorage and tokenserver integration tests"
+    )
+    parser.add_argument(
```

Comment:
I began the process of updating this file to allow us to run different test suites using command line flags. We'll need to do this in order to selectively run test suites in our CI pipelines for Syncstorage and Tokenserver